### ISSUE TITLE
Persistent Cookie module not working

### ISSUE THREAD

**Issue by lscorcia** [OpenIdentityPlatform/OpenAM#267]
**Describe the bug**
In latest sources, the persistent cookie module does not seem to work correctly. The `session-jwt` cookie is correctly created, but seems to not be used during login - worse, the login process asks the the user to login twice. I have followed the documentation available at https://backstage.forgerock.com/docs/openam/13.5/admin-guide/#persistent-cookie , which has not changed from the current implementation.

**To Reproduce**
1. Configure the auth chain as follows: `Persistent Cookie (Sufficient)`, `Data Store (Required)`
2. Configure the Persistent Cookie Module according to your environment (i.e. remove the `Secure` flag if testing locally and the `HttpOnly` flag if using XUI)
3. Logout and then login with the `demo` user (pw `changeit`)
4. See error 1: OpenAM asks for login twice
5. Enter credentials again. When at the dashboard, check in the browser dev tools that a `session-jwt` cookie is set
6. Close all browser windows and reopen OpenAM
7. See error 2: OpenAM asks for login instead of going directly to the dashboard

**Expected behavior**
The user should enter the credentials only once, and shouldn't need to login again when the Persistent Cookie is set.

**Additional context**
I have tried reverting all of the modifications applied to the Persistent Cookie module in this fork (`git checkout 3a049b1ecdbd5b36e0e08eec0f3343e80e6971ce ./openam-authentication/openam-auth-persistentcookie/src`) and after that the module works correctly, so this narrows the problem down to a handful of commits.
Probably some of the edits have been applied in the past to work around #237, but now they're no longer needed. There's little context about those commits so I don't really know what they were supposed to fix but I will try to take a look at them.
